fpga/Makefile

```diff
@@ -0,0 +1,124 @@
+#########################################################################################
+# fpga prototype makefile
+#########################################################################################
+
+#########################################################################################
+# general path variables
+#########################################################################################
+base_dir=$(abspath ..)
+sim_dir=$(abspath .)
+
+# do not generate simulation files
+sim_name := none
+
+#########################################################################################
+# include shared variables
+#########################################################################################
+include $(base_dir)/variables.mk
+
+# default variables to build the arty example
+SUB_PROJECT       := fpga
+SBT_PROJECT       := freedomPlatforms
+MODEL             := E300ArtyDevKitFPGAChip
+VLOG_MODEL        := E300ArtyDevKitFPGAChip
+MODEL_PACKAGE     := sifive.freedom.everywhere.e300artydevkit
+CONFIG            := E300ArtyDevKitConfig
+CONFIG_PACKAGE    := sifive.freedom.everywhere.e300artydevkit
+GENERATOR_PACKAGE := chipyard
+TB                := none # unused
+TOP               := E300ArtyDevKitPlatform
+
```

Comment:
Should the goal be to eventually eliminate most of these overrides? I believe eventually everything can just live as part of the core chipyard generator.

Reply:
I think thats related to the other checkbox in my review of the original PR. I believe the answer is yes.

Reply:
Yes. The goal is to get rid of these. I just hard-coded them so that this particular example works all the time. Ideally, this will change with more configuration exposed.